### DISCUSSION TITLE
calling its on a response body with a nested array doesn't recognize index 0

### DISCUSSION THREAD

**Discussion by klquam** [cypress-io/cypress#17793] [Questions and Help]
I have an alias on an intercept similar to the one below:

`cy.wait('@createUser').its('response.body.users[0].username').should('equal', 'awesome_user')`

to which I get an error message from cypress stating:

> cy.its() errored because the property: users[0] does not exist on your subject.

and when I rephrase the assertion to:

```
cy.intercept('GET', 'users', (req) => {
     req.reply((res) => {
            expect(res.body.users[0].username).to.equal('awesome_user')
     })
})
```

it passes.

My question is, is there a way to write that assertion from the alias?

**Comment by bahmutov** [comment]
Should be `.its('response.body.users.0.username')`

**Reply by klquam** [reply]
This is the answer, but for some reason I am unable to mark it as such =(
